stack/adding-a-new-package.rst

```diff
@@ -35,29 +44,40 @@ Replace ``afw`` with the relevant package name to get to the correct page on Git
 .. image:: /_static/build-ci/github_merge_button_settings.png
 
 2. Configure ``master`` branch to enable protections.
-For ``afw`` this is located at https://github.com/lsst/afw/settings/branches/master and can also be found from the "Branches" sidebar item on the settings screen.
-Select the "Protect this branch" option and this will make available additional checks.
-You should also enable status checks and require that branches be up to date before merging.
-With this enabled people will be able to use the GitHub merge button on Pull Requests and know that the :ref:`standard process <workflow-code-review-merge>` is being adhered to.
+For ``afw`` this is located at https://github.com/lsst/afw/settings/branches/ and can also be found from the "Branches" sidebar item on the settings screen.
+In the "Branch protection rules" section of that page you will have to click on "Add rule" to create a rul for ``master``.
+Add in ``master`` as the branch name pattern and then enable status checks and require that branches be up to date before merging.
+Administrators must be included in these protections since it's all to easy to make a mistake without realizing you have special override powers.
```

Comment:
```suggestion
Administrators must be included in these protections since it's all too easy to make a mistake without realizing you have special override powers.
```